<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addToMortos</class>
 <widget class="QDialog" name="addToMortos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>258</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="idMortosLabel">
       <property name="text">
        <string>Id do gado:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="idMortosComboBox"/>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="dataMorteLabel">
       <property name="text">
        <string>Data de morte:</string>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="dataMorteDateTimeEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>17</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QLabel" name="descricaoLabel">
       <property name="text">
        <string>Descircao: </string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="descricaoPlainTextEdit"/>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="adicionarPushButton">
       <property name="text">
        <string>Adicionar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="limparPushButton">
       <property name="text">
        <string>Limpar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="cancelarPushButton">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
